multicluster_setup.adoc

```diff
@@ -0,0 +1,143 @@
+= Multi-Cluster Setup
+
+Prerequisites:
+
+You need to boostrap 2 clusters and make sure you have a `kubeconfig` file for each cluster in a dedicated/separate location.
+
+== Setting up the KUBECONFIG
+
+First, let's setup a single `KUBECONFIG` with multiple contexts. Assuming that `$HOST_CONFIG` is the path to the `KUBECONFIG` file for your *Host* cluster, and `$MEMBER_CONFIG` is the path to the `KUBECONFIG` file for your *Member* cluster:
```

Comment:
Just my opinion: I like more the variables with the curly brackets - it's easier to spot them and it's more broadly used
```suggestion
First, let's setup a single `KUBECONFIG` with multiple contexts. Assuming that `${HOST_CONFIG}` is the path to the `KUBECONFIG` file for your *Host* cluster, and `${MEMBER_CONFIG}` is the path to the `KUBECONFIG` file for your *Member* cluster:
```
this applies for the rest of the doc

Reply:
I thought `$...` without curly brackets was easy to spot, but I can make the change if you think it's better for you.

Reply:
done in 71f5e35

Reply:
This looks much better, thanks :+1: